src/main/resources/templates/db_scripts/oaipmh/createOaiPmhViewFunction.sql

```diff
@@ -18,20 +18,22 @@ create or replace function ${myuniversity}_${mymodule}.getElectronicAccessName(v
 $$
 select jsonb_agg(distinct e)
```

Comment:
schema.json says:
```
 {
      "run": "after",
      "snippetPath": "oaipmh/createOaiPmhViewFunction.sql",
      "fromModuleVersion": "19.2.0"
    },
    {
      "run": "after",
      "snippetPath": "removeOldPrecedingSucceedingTitles.sql",
      "fromModuleVersion": "19.2.0"
    }
```
These .sql files will not be executed when upgrading from 19.2.0 to the current version. Please update fromModuleVersion.

Reply:
This will be release in the next version of mod-inventory-storage, 19.2.4, is it ok to set 12.2.4 in from module version?

Reply:
@mrdpopov Personally, I think the `fromVersion` in the mainline should be `19.3.0` and `19.2.4` in the bug fix back port.

@julianladisch is that a sensible approach?

Reply:
> These .sql files will not be executed when upgrading from 19.2.0 to the current version. Please update fromModuleVersion.

Is this because `fromModuleVersion` in this case means apply this script when the module is upgraded from something before this version to either that version or newer? 

And hence, an upgrade from 19.2.x to a version with this code in would not execute this script?

Reply:
I cant upgrade to the unreleased 19.0.4 now

Reply:
Using 19.2.4 in both mainline and bug fix is correct.
When upgrading from 19.2.3 to 19.3.1 it should run.
When upgrading from 19.2.4 to 19.3.0 it should not run again because it has run in 19.2.4.

Reply:
> Using 19.2.4 in both mainline and bug fix is correct.

Why would we want a version referred to on the mainline that should never be released from the mainline?

Is this to avoid it being executed twice if the bug fix release has previously been installed?

Reply:
> It looks like I cant put the unreleased 19.2.4 now in frommoduleversion because the build will fail.

Branch MODINVSTOR-527 with
```
     {
       "run": "after",
       "snippetPath": "oaipmh/createOaiPmhViewFunction.sql",
      "fromModuleVersion": "19.2.4"
     },
```
successfully builds.

Reply:
>Is this to avoid it being executed twice if the bug fix release has previously been installed?

Yes.

Reply:
Thanks @julianladisch for explaining this